Topic4Services/src/cartServices/ShoppingCartProxy.java

```diff
@@ -0,0 +1,38 @@
+package cartServices;
+
+import items.Item;
+
+import java.util.List;
+
+import payments.Payment;
+
+public class ShoppingCartProxy implements ShoppingCartService {

```

Comment:
i saw it in the example , and i thought that it ll be useful for the final project !